docs/products/postgresql/concepts/pg-connection-pooling.rst

```diff
@@ -0,0 +1,92 @@
+        classDef style_pg_services fill:#90ee90,stroke:#333,stroke-width:0px;
+        classDef style_client fill:lightgray,stroke:#333,stroke-width:0px;
+        pg_client_1(PG client) <-.->|Client establishing a new connection| postmaster
+        pg_client_2(PG client) ---> pg_backend_1(PG Backend 1)
+        pg_client_3(PG client) ---> pg_backend_2(PG Backend 2)
+        pg_client_4(PG client) ---> pg_backend_3(PG Backend 3)
+        pg_client_5(PG client) --->|Existing client connections| pg_backend_4(PG Backend 4)
+        pg_client_6(PG client) ---> pg_backend_5(PG Backend 5)
+        pg_client_7(PG client) ---> pg_backend_6(PG Backend 6)
+
+    subgraph PostgreSQL server
+        postmaster & pg_backend_1 & pg_backend_2 & pg_backend_3 & pg_backend_4 & pg_backend_5 & pg_backend_6
+    end
+
+    subgraph Clients
+        pg_client_1 & pg_client_2 & pg_client_3 & pg_client_4 & pg_client_5 & pg_client_6 & pg_client_7
+    end
+    class pg_client_1 style_main_node;
+    class pg_backend_1,pg_backend_2,pg_backend_3,pg_backend_4,pg_backend_5,pg_backend_6,postmaster style_pg_services;
+    class Clients style_client;
+
+Adding a PGBouncer pooler that utilizes fewer backend connections frees up server resources for more important uses, such as disk caching:
+
+.. mermaid::
+
+    graph LR
+        classDef style_main_node fill:orange,stroke:#333,stroke-width:0px;spacing:0px;
+        classDef style_pg_services fill:#90ee90,stroke:#333,stroke-width:0px;
+        classDef style_client fill:lightgray,stroke:#333,stroke-width:0px;
+        pg_client_1(PG client) <-.->|Client establishing a new connection| pgbouncer
+        pg_client_2(PG client) ---> pgbouncer
+        pg_client_3(PG client) ---> pgbouncer
+        pg_client_4(PG client) ---> pgbouncer
+        pg_client_5(PG client) --->|Existing client connections| pgbouncer
+        pg_client_6(PG client) ---> pgbouncer
+        pg_client_7(PG client) ---> pgbouncer
+        pgbouncer --> postmaster
+        pgbouncer --> pg_backend_1(PG Backend 1)
+        pgbouncer --> pg_backend_2(PG Backend 2)
+        pgbouncer --> pg_backend_3(PG Backend 3)
+        pgbouncer --> pg_backend_4(PG Backend 4)
+
+
+    subgraph PostgreSQL server
+        pgbouncer
+        postmaster & pg_backend_1 & pg_backend_2 & pg_backend_3 & pg_backend_4
+    end
+
+    subgraph Clients
+        pg_client_1 & pg_client_2 & pg_client_3 & pg_client_4 & pg_client_5 & pg_client_6 & pg_client_7
+    end
+    class pg_client_1 style_main_node;
+    class pg_backend_1,pg_backend_2,pg_backend_3,pg_backend_4,pg_backend_5,pg_backend_6,postmaster,pgbouncer style_pg_services;
+    class Clients style_client;
+
+Many frameworks and libraries (ORMs, Django, Rails, etc.) support client-side pooling, which solves much the same problem. However, when there are many distributed applications or devices accessing the same database, a client-side solution is not enough.
```

Comment:
rather than "a client-side solution is not enough", the negative way, instead let's say "a server-side solution is a better approach" - a positive take on the same thing.

Reply:
Fixed